### PR TITLE
Embedding existing jmx integrations with git submodule

### PR DESCRIPTION
**Pull request by tylerbenson**
Now uses a git submodule to load the integrations from `integrations-core`.
Will depend on a new jmxfetch release with https://github.com/DataDog/jmxfetch/pull/205 before this will work.

### CODE REVIEW

File: dd-java-agent/agent-jmxfetch/src/main/resources/datadog/trace/agent/jmxfetch/metricconfigs.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+activemq.yaml
```
Comment (mar-kolya) [inline dd-java-agent/agent-jmxfetch/src/main/resources/datadog/trace/agent/jmxfetch/metricconfigs.txt]: Could you please add readme file in this directory explaining its contents and where those files are coming from?

Comment (tylerbenson) [reply dd-java-agent/agent-jmxfetch/src/main/resources/datadog/trace/agent/jmxfetch/metricconfigs.txt]: Done.  Does the new readme look ok?

**Comment by mar-kolya** [comment]
LGTM

**Comment by tylerbenson** [comment]
Can't merge this until we get a new version of jmxfetch.

**Comment by tylerbenson** [comment]
@realark That was fully my intention.  This was just intended to demonstrate the flow after those files were brought in.  Sorry if that wasn't more clear.

**Comment by realark** [comment]
@tylerbenson Oh sure. I'll remove my review in that case.

File: dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle [hunk 2]
```diff
@@ -27,3 +27,19 @@ shadowJar {
 jar {
   classifier = 'unbundled'
 }
+
+task submodulesUpdate(type:Exec) {
+  group 'Build Setup'
+  description 'Initializes and updates integrations-core git submodule'
+  commandLine 'git', 'submodule', 'update', '--init', 'integrations-core'
```
Comment (tylerbenson) [inline dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle]: Update here just means populating the directory with the repo contents, it doesn't change the commit the submodule is based off of.

**Comment by tylerbenson** [comment]
@mar-kolya This is ready for review now that we have the proper version of jmxfetch.

**Pull request merged**